### ISSUE TITLE
GPL-878-16 - As a user I want a pooling page

### ISSUE THREAD

**Issue by stevieing** [sanger/traction-ui#638]
**User story**
As a user I want a pooling page

**Who are the primary contacts for this story**
e.g. John S (don't include surnames in public repos)

**Acceptance criteria**
To be considered successful the solution must allow:
- [x] It should display a list of all registered pools
- [x] For each pool it should show
  - [x] The tube barcode
  - [x] The pool volume, concentration and fragment size
  - [x] A show details button to display samples in the pool
  - [x] A link to the pool edit page

Nice to haves (Adding here in case any are dirt cheap):
- [x] Allow filtering by included samples (I *think* this one is free)
- [ ] Show if a pool has been sequenced

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**
